app/src/main/java/com/project/ianime/utils/image/ImageStub.kt

```diff
@@ -10,7 +10,7 @@ class ImageStub {
      * Get encoded image bitmap with Base-64
      * @param [bitmap] - bitmap of current upload image
      */
-    fun getEncodedSelectedImageBitmap(bitmap: Bitmap): String{
+    fun getEncodedSelectedImageBitmap(bitmap: Bitmap): String {
```

Comment:
it's no longer a base64 str, it's binary file now

Reply:
this file will be removed, no need